src/components/memoBlock/memo.component.tsx

```diff
@@ -5,6 +5,7 @@ import { getImages } from "store/getImages";
 import { Props, Image } from "components/interfaces";
 import { RootState } from "store/imagesSlice";
 import Blocks from "components/blocks";
+import Title from "components/title/index";
```

Comment:
```suggestion
import Title from "components/title";
```

You can be shorter in the importing here as it's the index file.